composer.json

```diff
@@ -11,12 +11,16 @@
         }
     ],
     "require": {
-        "elasticsearch/elasticsearch": "^2.1|^5.0|^6.0|^7.0",
-        "aws/aws-sdk-php": "^3.0"
+        "aws/aws-sdk-php": "^3.0",
+        "ezimuel/ringphp": "^1.1.2"
```

Comment:
Was this dependency used anywhere?

Reply:
Ah. I started out with just copying the defaultHandler method. That method uses classes from that package which made me require it. 

With the current solution of checking which library is used and just delegating to that method, there is no dependency anymore. Except for the tests, which still uses one of the classes from that library. 

So I could move it to require-dev. Or still copy the defaultHandler code which might make it cleaner.

Reply:
Could you move it to `require-dev`? This is a fork of guzzlehttp/ringphp that is maintained by Elasticsearch, so it's not guaranteed that Opensearch-PHP would stick with it. It's unclear what would be used instead, but it seems best to let (Open|Elastic)search-PHP bring its own HTTP handler in case they diverge in the future.

Reply:
Done. Thx!